### PR TITLE
tests: print user sessions info on debug-each

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This info could be useful in case of errors related to user sessions

### CODE REVIEW

**Review by anonymouse64 — APPROVED** [review]
lgtm thanks for this

**Comment by anonymouse64** [comment]
note that the log for the snapd user session agent is stunningly quiet unfortunately, but as we add things to the user session agent which make it more talkative having this will let us actually see that output :smile:

File: spread.yaml [hunk 1]
```diff
@@ -444,6 +444,11 @@ debug-each: |
 
         echo '# journal messages for snapd'
         "$TESTSTOOLS"/journal-state get-log -u snapd
+
+        echo '# user sessions information'
+        journalctl --user -u snapd.session-agent.service
+        systemctl status --user snapd.session-agent  || true
```
Comment (bboozzoo) [inline spread.yaml:450]: Keep in mind that wold print the session agent in the root's session, but at some point we may be interested in the sssion agent of the test user too.

**Review by bboozzoo — APPROVED** [review]

**Comment by bboozzoo** [comment]
@mvo5 this can land now.

**Pull request merged**